data/hai/hai reveal 4 middle.txt

```diff
@@ -72,9 +72,9 @@ mission "Hai Reveal [A02] Symbol Identification"
 				`	(Walk into the cave.)`
 				`	(Come back later.)`
 					defer
-			`	You walk twenty meters into the cave and enter an open doorway before passing through a presently unmanned security post into a small and roughly designed underground facility. There's a variety of medical machinery in rooms with large glass windows immediately to the left, and several computers arrayed around a makeshift living area with what looks to be some sleeping quarters off to the back and right. Some of the people you've rescued are present, though there are many more humans and Hai that you haven't met. A few of them have pulse rifles, but your attention is quickly captured by a woman walking towards you.`
+			`	You walk twenty meters into the cave and enter an open doorway before passing through an unmanned security post into a small and roughly designed underground facility. There's a variety of medical machinery in rooms with large windows immediately to the left, and computers arrayed around a makeshift living area with sleeping quarters off to the right. Some people you've rescued are present, with many humans and Hai you haven't met. Some have pulse rifles, but your attention is quickly captured by a woman walking towards you.`
 			`	"Greetings Captain <last>, my name is Teeneep Goot'r. I'm a social advocate among the Hai, and I'm trying to help your friends with a problem. As you know, they were kidnapped by humans. A few other Hai and human victims are here; they were mostly rescued by other kind merchants in human space, but around half of those here are my staff."`
-			`	"My organization has been researching the issue of disappearing Hai ever since some humans also began disappearing here under similar circumstances, and we've discovered some patterns to the disappearances. Perhaps more pertinently though, since we've found those patterns we've been able to reach some of the victims more quickly upon their return and even rescue a couple ourselves. However, those Hai that you rescued and several others, too, have a very strange commonality. These kidnap victims are all experiencing mild cadmium poisoning."`
+			`	"My organization has been researching the disappearances of Hai and humans, and we've discovered some patterns. Since we've found those patterns, we've been able to reach some of the victims more quickly upon their return and even rescue some ourselves. However, those Hai that you rescued and several others, too, have a strange commonality: mild cadmium poisoning."`
```

Comment:
```suggestion
			`	"My organization has been researching the disappearance of Hai ever since humans began disappearing here under similar circumstances. We've discovered some patterns to these abductions, and through those we've been able to reach some of the victims more quickly upon their return, and even rescue some ourselves. But we have noticed something strange among the abductees: those Hai that you rescued, and several others, too, are all experiencing mild cadmium poisoning."`
```
Even though the last one was wordy, I don't want to lose all the context. I can't remember if I was told off for putting that last comma there before.